### PR TITLE
[REF] mail: remove old chatter template

### PR DESCRIPTION
**Pull request by xdu-odoo**
task-2246809

### CODE REVIEW

**Review by alexkuhn — APPROVED** [review]
happy weight loss

**Comment by xdu-odoo** [comment]
@alexkuhn Since I modified something in here, you probably should review it again even if you approved it before

File: addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js [hunk 1]
```diff
@@ -31,6 +31,13 @@ FormRenderer.include({
         // Do not load chatter in form view dialogs
         this._isFromFormViewDialog = params.isFromFormViewDialog;
     },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        this._destroyChatterContainer();
```
Comment (alexkuhn) [inline addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js]: Apparently `.destroy()` is automatically called on wrapper when parent `.destroy()` is called.
The only important parts are setting `undefined` on `_chatterContainerComponent` and `.off('o_chatter_rendered')`.

This method should be called only when there is a `chatterContainerComponent`, it's weird to have the `if` condition inside the `_destroyChatterContainer()` function. Also I wouldn't bother with another function if it's called only in `destroy()` and just dump the 2 expressions there inside a `if` conditional branch.

**Comment by xdu-odoo** [comment]
Green :muscle:

**Review by seb-odoo — APPROVED** [review]
I'm not 100% sure what the destroy change is about, but it doesn't bother me so ok for me if it's ok for you 2.

**Comment by seb-odoo** [comment]
Not `web_studio` in community commit though, it's just `mail` module

**Review by alexkuhn — APPROVED** [review]

**Pull request merged**